cpp/tests/test_epi_data_io.cpp

```diff
@@ -94,6 +94,36 @@ TEST(TestEpiDataIo, read_rki_error_age)
     ASSERT_THAT(print_wrap(result), IsFailure(mio::StatusCode::InvalidValue));
 }
 
+TEST(TestEpiDataIo, read_confirmed_cases_noage)
+{
+    Json::Value js(Json::arrayValue);
+    js[0]["Date"]      = "2021-12-01";
+    js[0]["Confirmed"] = 1;
+    js[0]["Deaths"]    = 2;
+    js[0]["Recovered"] = 3;
+
+    js[1]["Date"]      = "2021-12-02";
+    js[1]["Confirmed"] = 4;
+    js[1]["Deaths"]    = 5;
+    js[1]["Recovered"] = 6;
+
+    auto result = mio::deserialize_confirmed_cases_noage(js);
+    ASSERT_THAT(print_wrap(result), IsSuccess());
+
+    auto rki_data_noage = result.value();
+    ASSERT_EQ(rki_data_noage.size(), 2);
+
+    ASSERT_EQ(rki_data_noage[0].date, mio::Date(2021, 12, 1));
+    ASSERT_EQ(rki_data_noage[0].num_confirmed, 1);
+    ASSERT_EQ(rki_data_noage[0].num_deaths, 2);
+    ASSERT_EQ(rki_data_noage[0].num_recovered, 3);
+
+    ASSERT_EQ(rki_data_noage[1].date, mio::Date(2021, 12, 2));
+    ASSERT_EQ(rki_data_noage[1].num_confirmed, 4);
+    ASSERT_EQ(rki_data_noage[1].num_deaths, 5);
+    ASSERT_EQ(rki_data_noage[1].num_recovered, 6);
```

Comment:
Why do you test this for two days and not just one?

Reply:
In the other tests, we also tested more than one day. I also added two dates to the Json::Value to test the size. Maybe the asserts aren't necessary, but it is also not a bad thing to keep them?